pkg/apis/kops/cluster.go

```diff
@@ -312,6 +312,8 @@ type EtcdClusterSpec struct {
 	LeaderElectionTimeout *metav1.Duration `json:"leaderElectionTimeout,omitempty"`
 	// HeartbeatInterval is the time (in milliseconds) for an etcd heartbeat interval
 	HeartbeatInterval *metav1.Duration `json:"heartbeatInterval,omitempty"`
+	// ContainerImage is the etcd docker image to use. Setting this will ignore the Version specified.
+	ContainerImage string `json:"containerImage,omitempty"`
```

Comment:
We should match the names we are using in componentconfig.go

Image

ContainerImage is kinda redundant.